src/presentation/http/router/user.test.ts

```diff
@@ -17,13 +29,10 @@ describe('User API', () => {
 
       expect(response?.statusCode).toBe(200);
 
-      const body = response?.json();
-
-      expect(body).toStrictEqual({
-        'id': '1',
-        'email': 'a@a.com',
-        'name': 'Test user 1',
-        'photo': '',
+      expect(response?.json()).toMatchObject({
+        name: user.name,
+        email: user.email,
+        photo: '',
```

Comment:
why not `user.photo`?

Reply:
Because `user.photo` is null - we don't pass it, but the api returns just ''